### DISCUSSION TITLE
When duplicating an entry Craft should always set it to OFFLINE/DISABLED on all sites

### DISCUSSION THREAD

**Discussion by bencresty** [craftcms/cms#11986] [Ideas] [answered]
When duplicating an entry Craft doesn't turn off the ONLINE/LIVE/ENABLED switch on the sites the original is currently live at. Meaning that there's no way to create a duplicate without at least a short period of time the duplicate to be visible publicely to visitors of the frontend, while obviously the duplicate entry needs editing before it's ready to go live.

I can't think of any scenario when we would like to have a duplicated entry to go live without any change to the content. Also it's prone to errors as it's easy to forget to turn off the ONLINE/LIVE/ENABLED switches, so when forgetting you're editing a page while it's already live for the public/google/bookmarking with wrong urls (because there's no right slug set on the duplicated entry etc.)

**Please make Craft automatically disable/set offline duplicated entries before saving them!**

**Comment by brandonkelly** [comment, marked as answer]
This was previously requested at #10244 – you can follow along there for updates.
